src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -8,7 +8,7 @@ import { FileTree } from '@astrojs/starlight/components';
 
 Astro is designed to make it easy to deploy to any cloud provider for SSR (server-side rendering). This ability is provided by __adapters__, which are [integrations](/en/reference/integrations-reference/). See the [SSR guide](/en/guides/on-demand-rendering/) to learn how to use an existing adapter.
 
-## What is an adapter
+## What is an Adapter
```

Comment:
The next heading uses a capitalized `adapter` so... either both are capitalized or neither.